.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,53 @@
+name: Update gem RBIs on Dependabot PRs
+
+on: workflow_call
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-gem-rbis:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: ${{ github.actor == '"dependabot[bot]"' }}
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: ruby/setup-ruby@v1
+        id: setup
+        with:
+          bundler-cache: true
+          ruby-version: 3.2
```

Comment:
This should be customizable

Reply:
What do we think about committing a `.ruby-version` for public repos? Then we can remove this and settig and it will pick that up by default.

Reply:
Yes we can use that to customize it. It'd have to be highlighted in the docs since it's a mandatory onboarding step.

I don't think removing the setting is easiest. We can keep the setting so that we have a logic for looking at `.ruby-version` and then falling back to reading `dev.yml` if it doesn't exist. When a new shopify repo onboards it means one less change to remember.

Reply:
In my opinion, the Ruby version should be configurable. If users of the workflow want to read `.ruby-version` or any other configuration file, then they should be able to do that from their usage of the workflow.

Reply:
For now I've updated it to read from `dev.yml`, similar to how the internal action works. We can later consider options for supporting other configurations. It would nice to have it consistent with the [options available through setup-ruby.](https://github.com/ruby/setup-ruby#supported-version-syntax)